client/src/components/NotificationDisplay/NotificationDisplay.tsx

```diff
@@ -0,0 +1,67 @@
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+import Menu from '@material-ui/core/Menu';
+import { Box } from '@material-ui/core';
+import { useNotification } from '../../context/useNotificationContext';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+import { Badge } from '@material-ui/core';
+
+const NotificationLink = (): JSX.Element => {
+  const classes = useStyles();
+  const { notifications, update } = useNotification();
+  console.log(`notifications`, notifications);
```

Comment:
This console log is not needed here. Please remove it.

Reply:
Gone.